paasta_tools/chronos_tools.py

```diff
@@ -185,6 +186,13 @@ def __eq__(self, other):
             (self.branch_dict == other.branch_dict)
         )
 
+    def __normalize(self):
+        """When the config supports alternative syntaxes, convert them
+        to a normal form."""
+        parents = self.get_parents()
+        if parents is not None and not isinstance(parents, list):
+            self.config_dict['parents'] = [parents]
```

Comment:
For now, junk the `eq` method. We don't use it and we don't normalize all the other things. Stick with a get_parents getter like most of our other code so that get_parents() always returns something consistent.